README.md

```
@@ -42,7 +42,7 @@ me.say_hi()
 
 ### Frameworks & Libraries
 <p>
-  <img src="https://skillicons.dev/icons?i=fastapi,django,flutter" />
+  <img src="https://skillicons.dev/icons?i=odoo,fastapi,django,flutter" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)